include/glow/Support/Random.h

```diff
@@ -16,8 +16,51 @@
 #ifndef GLOW_SUPPORT_RANDOM_H
 #define GLOW_SUPPORT_RANDOM_H
 
+#include <random>
```

Comment:
<random>  will be included into all of the files that include support/random.h. Is there a way to move <random> into the cpp file somehow?

Reply:
oops, github removes words in \< \> such as \<random\>

Reply:
Ugh. I tried, but I couldn't find a way.

The problem is the `Handle::randomize()` methods. They are templates, so they need to be defined inline. They need to call methods on the `PseudoRNG` class, so they need the full definition of that class; a forward decl won't do. The full class definition contains the Mersenne twister engine as a member, so it needs the `<random>` header.

I didn't look into pimpl patterns and stuff like that because it seems like C++ really just wants you to include all the things everywhere now. It could probably be made to work with some extra indirections and memory allocations.